packages/base/source/server/events.js

```diff
@@ -5,7 +5,7 @@ Space.messaging.define(Space.messaging.Event, 'Todos', {
   },
 
   TodoCreated: {
-    id: Guid,
+    todoId: Guid,
```

Comment:
At some point we did this:
https://github.com/meteor-space/todos/commit/6c1c4fd74637222fde1b11cae88276031aad0a0d

Because Dominik had this point: 
>i would change id to todoId so that it's clear what it is targeting inside the list"

TodoCreated event was the only one that was not modeled like that so I fixed it. 

In context of the aggregate that receives the command, and domain logic code, it makes sense to be explicit and leave `todoId` and use `id` for instance of the aggregate.